experimenter/experimenter/experiments/tests/api/v5/test_queries.py

```diff
@@ -2722,4 +2727,35 @@ def assertChoices(data, text_choices):
             self.assertIn(
                 {"id": str(project.id), "name": project.name}, config["projects"]
             )
-        self.assertEqual(config["user"], user_email)
+        self.assertEqual(config["user"], self.user_email)
+
+    @parameterized.expand(
+        [
+            object(),
+            123,
+            json.dumps(
+                {
+                    "bogus": "data",
+                }
+            ),
+        ]
+    )
```

Comment:
Probably unnecessary, but could also include a `SampleSizes` instance here since that's what caused the original issue.